app/tests/unit/specs/components/account/AccountMenu.spec.js

```diff
@@ -7,14 +7,17 @@ describe(`AccountMenu`, () => {
   beforeEach(() => {
     wrapper = shallowMount(AccountMenu, {
       props: {
-        address: `cosmos1234`,
-        networkId: `cosmos-hub-mainnet`,
+        account: {
+          address: `cosmos1234`,
+          networkId: `cosmos-hub-mainnet`,
+          sessionType: `local`,
+        },
       },
       stubs: [`router-link`],
     })
   })
 
-  it(`should show the AccountMenu page`, async () => {
+  it.skip(`should show the AccountMenu page`, async () => {
```

Comment:
?

Reply:
yeah, it fails. I don't understand why. Please help

Reply:
next time, tell me and not just leave it in ;)

Reply:
yeah, my intention was telling, not hiding it :sweat_smile: I just had to leave at the time